src/utils/raw.rs

```diff
@@ -131,7 +131,7 @@ pub fn append_file(dest: &Path, line: &str) -> io::Result<()> {
     Ok(())
 }
 
-pub(crate) fn symlink_dir(src: &Path, dest: &Path) -> io::Result<()> {
+pub fn symlink_dir(src: &Path, dest: &Path) -> io::Result<()> {
```

Comment:
I had to change this from `pub(crate)` to `pub` to call it, I hope that's not a problem.

Reply:
I think it's okay, the `rustup` lib API is currently not intended for third-party usages anyway.